src/linear_algebra/quadratic_form.lean

```diff
@@ -436,6 +436,19 @@ begin
   rw [mul_right_comm, mul_inv_of_self, one_mul],
 end
 
+-- Change to module over rings once #6585 is merged
+lemma exists_quadratic_form_neq_zero [nontrivial M]
```

Comment:
I'm used to referring to vectors `x` with `Q x = 0` as null vectors. If this naming is normative, then I'd suggest the name
```suggestion
lemma exists_non_null [nontrivial M]
```

Reply:
I believe mathlib style is that if a term (such as `null_vector`) has not yet been made a `def`, then it is not used in lemma names.  And we don't have enough material yet to justify making that definition.  On the other hand, I think it is definitely worthwhile adding a docstring mentioning "null vector" to both this and the bilinear form version.